pcapfile/savefile.py

```diff
@@ -89,7 +89,7 @@ def _load_savefile_header(file_h):
         raw_savefile_header = file_h.read(24)
     except UnicodeDecodeError:
         print("\nMake sure the input file is opened in read binary, 'rb'\n")
-        raise
+        raise InvalidEncoding("Could not read file. Might not be opened as binary.")
```

Comment:
I propose changing this to message to `"Could not read file; it might not be opened in binary mode."